toss_4_x86_64_ib_cray/packages.yaml

```diff
@@ -112,15 +122,21 @@ packages::
       prefix: /opt/rocm-5.4.3/
     - spec: rocprim@5.5.1
       prefix: /opt/rocm-5.5.1/
+    - spec: rocprim@5.6.0
+      prefix: /opt/rocm-5.6.0/
   cray-mpich:
     buildable: false
     externals:
-    - spec: cray-mpich@8.1.25%cce@15.0.1
-      prefix: /usr/tce/packages/cray-mpich/cray-mpich-8.1.25-cce-15.0.1-magic
+    - spec: cray-mpich@8.1.26%cce@15.0.1
+      prefix: /usr/tce/packages/cray-mpich/cray-mpich-8.1.26-cce-15.0.1-magic
+    - spec: cray-mpich@8.1.26%cce@16.0.0
+      prefix: /usr/tce/packages/cray-mpich/cray-mpich-8.1.26-cce-16.0.0-magic
     - spec: cray-mpich@8.1.25%gcc@12.1.1
-      prefix: /usr/tce/packages/cray-mpich/cray-mpich-8.1.25-gcc-12.1.1-magic
+      prefix: /usr/tce/packages/cray-mpich/cray-mpich-8.1.26-gcc-12.1.1-magic
     - spec: cray-mpich@8.1.25%rocmcc@5.5.1
-      prefix: /usr/tce/packages/cray-mpich/cray-mpich-8.1.25-rocmcc-5.5.1-magic
+      prefix: /usr/tce/packages/cray-mpich/cray-mpich-8.1.26-rocmcc-5.5.1-magic
+    - spec: cray-mpich@8.1.25%rocmcc@5.6.0
+      prefix: /usr/tce/packages/cray-mpich/cray-mpich-8.1.26-rocmcc-5.6.0-magic
```

Comment:
I have started to see failures in tests builds due to missing appropriate MPI (with CCE). So I’m taking the habit to add one MPI per compiler.

Reply:
Sounds like a good idea.